### PR TITLE
Release updates to docs

### PR DESCRIPTION
**Pull request by jjenzz**
Updates docs to reflect changes in latest release.

**NOTE**: base branch is `release-update` to ease review. This should be merged into `main` after the base branch.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code.
- [ ] Include the URL where we can test the change in the body of your PR.

This pull request:

- [ ] Fixes a bug
- [X] Adds additional features/functionality
- [X] Updates documentation or example code
- [ ] Other

### CODE REVIEW

File: pages/primitives/docs/utilities/portal/0.0.4/index.mdx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default () => <Portal.Root>Content</Portal.Root>;
 
 ### Root
 
-Anything you put inside this component will be rendered in a separate `<radix-portal>` element. By default, this element will be appended to `document.body` but you can choose a different container by using the `containerRef` prop.
+Anything you put inside this component will be rendered in a separate `<div data-radix-portal>` element. By default, this element will be appended to `document.body` but you can choose a different container by using the `containerRef` prop.
```
Comment (benoitgrelard) [inline pages/primitives/docs/utilities/portal/0.0.4/index.mdx:42]: Should we mention about the styles we add?

Comment (jjenzz) [reply pages/primitives/docs/utilities/portal/0.0.4/index.mdx:42]: Do we need to? That feels like an implementation detail to me. I think people just expect it to be on top right?

Comment (benoitgrelard) [reply pages/primitives/docs/utilities/portal/0.0.4/index.mdx:42]: Yeah true, good point. We don't mention these things on other functional stuff.

File: pages/primitives/docs/components/slider/0.0.3/index.mdx [hunk 2]
```diff
@@ -263,7 +263,7 @@ The track that contains the `Slider.Range`.
 
 ### Range
 
-The range part. Must live inside `Slider.Track`.
```
Comment (benoitgrelard) [inline pages/primitives/docs/components/slider/0.0.3/index.mdx:266]: Oops…

File: pages/primitives/docs/components/tooltip/0.0.4/index.mdx [hunk 3]
```diff
@@ -423,10 +423,13 @@ export default () => (
 
 #### Implementation
 
+Use the [`Slot` utility](../../utilities/slot) to convert the trigger part into a slottable area. It will replace the trigger with the child that gets passed to it.
+
 ```jsx
 // your-tooltip.jsx
 import React from 'react';
 import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+import { Slot } from '@radix-ui/react-tooltip';
```
Comment (benoitgrelard) [inline pages/primitives/docs/components/tooltip/0.0.4/index.mdx:432]:
```suggestion
import { Slot } from '@radix-ui/react-slot';
```

File: yarn.lock [hunk 4]
```diff
@@ -972,6 +972,11 @@
   resolved "https://registry.yarnpkg.com/@radix-ui/react-polymorphic/-/react-polymorphic-0.0.3.tgz#741fd65de2ae4c97a014a58bc4a517cf4d55291d"
   integrity sha512-Fc5RfU8jvHYUUNCe+l6oeF3TN4SEXzijLkNRa/Y3qhUV3L0g9ASzOhyzfE11jdDRFUU6gDdoFI5ZAdvYBKEuLg==
 
+"@radix-ui/react-polymorphic@0.0.4":
```
Comment (benoitgrelard) [inline yarn.lock:975]: Looks like you need to run `yarn` perhaps?

File: pages/primitives/docs/components/slider/0.0.3/index.mdx (hunk 2 above)
Comment (jjenzz) [reply pages/primitives/docs/components/slider/0.0.3/index.mdx:266]: How on earth did that happen

Comment (jjenzz) [reply pages/primitives/docs/components/slider/0.0.3/index.mdx:266]: Hahahah, too trigger happy on the `CMD + D`

**Review by benoitgrelard — APPROVED** [review]

**Review by peduarte — APPROVED** [review]

**Pull request merged**